packages/shared/examples/shaped-scatter-plot/shaped-scatter-plot.component.html

```diff
@@ -0,0 +1,17 @@
+<h2>The Rise and Rise of A.I. Large Language Models</h2>
+<vis-bullet-legend [items]="legendItems"></vis-bullet-legend>
+<vis-xy-container [data]="data" [height]="600">
+  <vis-scatter
+    [x]="getX"
+    [y]="getY"
+    [color]="getColor"
+    [size]="15"
+    [shape]="getShape"
+    [label]="getLabel"
+    [labelPosition]="'bottom'"
```

Comment:
Not a big deal but for consistency's sake I would use this syntax: `labelPosition="bottom"`, like you have for the cursor property below it.

Reply:
True! I would just drop that line entirely then.

And `cursor="pointer"` is fine